### ISSUE TITLE
Wish: Design Script & Node Synergy

### ISSUE THREAD

**Issue by Amoursol** [DynamoDS/Dynamo#4787]
Would it be possible to have when you hover over a node the way in which you can call that functionality in DesignScript? 

Ie. I have no idea how to create a list in DesignScript, but if I hovered over the 'List.Create' node it would tell me how.

**Comment by monikaprabhu** [comment]
You can use AutoComplete to get the function signature - E.g. Start a CBN and type Point. and select ByCoordiantes. Then type "(" which will give you the function signature and overloaded methods available.
![autocomplete](https://cloud.githubusercontent.com/assets/5218721/8401537/7b558cb8-1e62-11e5-98e4-3e487e7b33a0.png)

**Comment by Amoursol** [comment]
Hello Monika,

Many thanks for replying! I do use the AutoComplete functionality (Awesome by the way), but I was more asking about DesignScript calls that don't logically transfer from nodes. Sorry if I wasn't more clear. 

Some of the nodes you have do have what functionality I was talking about, but others do not. Here is an example of how I would (With my very limited coding experience) try to guess at using List.Create in DesignScript:

![image](https://cloud.githubusercontent.com/assets/12857357/8403227/759a9da4-1e3a-11e5-977e-6307a7d0cfd6.png)

I do believe your "Node to Code", of which I only just found out about, will solve this issue however :)

**Comment by aparajit-pratap** [comment]
In the case of `List.Create` we do not have a DesignScript equivalent function since it is a UI node, i.e. it has UI elements such as `+` and `-` UI buttons.
![image](https://cloud.githubusercontent.com/assets/5710686/8403790/42d0e5fa-1e7a-11e5-8cd0-e85e69cc054a.png)
Therefore if you try converting the above nodes to code, you will see that only the nodes that have a DS equivalent (which is `List.Reverse` in this case) will be converted to code as seen below:
![image](https://cloud.githubusercontent.com/assets/5710686/8403804/518e5456-1e7a-11e5-8b45-c96e7b5e56f6.png)

**Comment by Amoursol** [comment]
Ok gotcha. Thanks for clarifying!

**Comment by aparajit-pratap** [comment]
But maybe we should start providing DS function equivalents for such UI nodes so that it is consistent with the rest of them.
@kronz, @riteshchandawar comments?

**Comment by kronz** [comment]
Yes!  Started a task for internal discussion: MAGN-7772 Implement DS function equivalents to UI nodes (R2)
